src/main/java/uk/gov/hmcts/reform/pip/data/management/database/ArtefactRepository.java

```diff
@@ -79,9 +79,9 @@ List<Artefact> findArtefactBySearch(@Param(SEARCH_TERM_PARAM) String searchTerm,
     @Query(value = "SELECT * FROM Artefact WHERE location_id LIKE '%NoMatch%'", nativeQuery = true)
     List<Artefact> findAllNoMatchArtefacts();
 
-    @Query(value = "SELECT cast(ARTEFACT_ID as text), display_from, display_to, language, "
-        + "provenance, sensitivity, source_artefact_id, type, content_date, location_id, list_type, "
-        + "cast(search as text) FROM artefact",
+    @Query(value = "SELECT cast(artefact_id as text), display_from, display_to, language, "
+        + "provenance, sensitivity, source_artefact_id, type, content_date, location_id, null, list_type "
```

Comment:
Is there any way to avoid the null column here?

Reply:
Have removed the null column from database. Instead I insert that extra empty column in PublicationService class. I think it is slightly better this way. There is no location_name field from the artefact table and the ticket wants a empty field to be returned in the excel table.

Reply:
Sounds good, thanks